src/js/components/filters.js

```diff
@@ -164,14 +159,14 @@ class Filters {
         all.innerHTML = 'ALL';
```

Comment:
Nit, and realise you didn't change this here, but do we ever use HTML here? If not could this be:

```suggestion
        all.textContent = 'ALL';
```

Better for performance and security.